src/main/java/uk/gov/digital/ho/hocs/info/domain/exception/ApplicationExceptions.java

```diff
@@ -76,24 +76,6 @@ public LogEvent getEvent() {
         }
     }
 
-    class StandardLineNotFoundException extends RuntimeException {
-        private final LogEvent event;
-
-        public StandardLineNotFoundException(String msg, LogEvent event, Object... args) {
-            super(String.format(msg, args));
-            this.event = event;
-        }
-
-        public StandardLineNotFoundException(String msg, Object... args) {
-            super(String.format(msg, args));
-            this.event = LogEvent.STANDARD_LINE_NOT_FOUND;
```

Comment:
Can this LogEvent be removed now from the enum?

Reply:
yep... thanks